client/src/components/widgets/ProvenanceCloseButton.vue

```diff
@@ -0,0 +1,43 @@
+<template>
```

Comment:
The vertical positions are different between the two files, `CloseButton.vue` is for the drill-down panel so when I used this component, the button overlapped with the "Close Simulation" button. Maybe I can incorporate this as a button inside `ProvenanceGraph.vue` to avoid having an extra component?

Reply:
Actually, you should be able to use the `deep` selector to modify the css for that `CloseButton` component. https://www.telerik.com/blogs/understanding-vue-deep-css-selector The idea is that you might need to apply specific styling for a specific component depending on the parent or where you're using it. @YohannParis can probably explain this better or propose other options but I think this should work without having to create a new component.

Reply:
Thanks for the pointer, I have used this method to modify the position of the `CloseButton` and removed the new component